### ISSUE TITLE
GearVR: Thread Priority Security Exception

### ISSUE THREAD

**Issue by mirage335** [y-fujii/ovrvnc#3]
GearVR support might be nice, considering the typical presence of a reasonably fast onboard cellular connection. Please consider.

Currently, attempting to run the APK results in a "Thread Priority Security Exception" error.
https://headjack.io/tutorial/solved-thread-priority-security-exception-gear-vr/

**Comment by y-fujii** [comment]
Yes, GearVR support would be nice.  However, in my understanding, installing .apk to GearVR device requires an osig (Oculus Signature File), which is tied to a specific device (See <https://developer.oculus.com/distribute/latest/concepts/publish-mobile-app-signing/>).  It means we need to generate osig **for each device** we use.

You may be able to use ovrvnc prebuilt apk to inject osig by yourself, but current ovrvnc probably has GearVR-specific issues, which may be difficult to fix because I don't have any GearVR devices (Pull Request is welcome).

**Comment by mirage335** [comment]
Ah, interesting. I might look into signing the apk. I am not really involved in Android or GearVR programming at the moment though, so it would be hard for me to generate a pull request.
